### DISCUSSION TITLE
CC fields in checkout block

### DISCUSSION THREAD

**Discussion by energizer-dev** [woocommerce/woocommerce#45013] [WooCommerce, Blocks, and Site Editing]
I am working on a payment plugin, but I am having issues displaying the credit card fields in the new checkout blocks.

What would be the best way to do this?

**Comment by angelamollina30** [comment]
Hello, I am having an issue with this too, can you share the code where you are having a problem so I can check on it too? Hope someone can help us too.

**Comment by opr** [comment]
Hi @energizer-dev, thanks for the post.

Do you have anything you can share with us so far? What specific errors (if any) are you facing when trying to integrate your payment method?

By the way, we have some documentation about integrating a payment method that may be useful to you https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/checkout-payment-methods/payment-method-integration.md#payment-methods---registerpaymentmethod-options-
